_content/excerpts/of-theras-and-theris_roy-kumkum.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Of Theras and Therīs: Visions of liberation in the Early Buddhist Tradition"
+authors:
+  - "KumKum Roy"
+drive_links:
+  - "https://drive.google.com/file/d/1k79UxZwl7sqXs94iAWzfrJ_1xdRW80OG/view?usp=sharing"
+status: featured
+course: tg
+tags:
+  - poetry
+  - feminism
+  - pali-canon
+year: 2010
+publisher: oxford
+address: "UK"
+booktitle: "The Power of Gender and the Gender of Power: Explorations in Early Indian History"
```

Comment:
It seems (from the above link) that this was actually an article first published in `Studies in Humanities and Social Sciences Vol. 5 No. 2 (1998)`? So perhaps this belongs in `articles` instead